src/gui/plotWidget.cpp

```diff
@@ -262,3 +262,38 @@ void Plot::clearSelection() {
         graph->setSelection(QCPDataSelection{});
     }
 }
+
+void Plot::convertToNewUnit(UnitValue current, UnitValue next) {
```

Comment:
Why have a `current` argument and a `Plot::lastUnit` at the same time?
I'd suggest using the `Plot::lastUnit` property.

Reply:
Removed current, changed to Plot::lastUnit